packages/components/tabs/src/tab-group.ts

```diff
@@ -277,7 +277,11 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
                   <sl-icon name="ellipsis" slot="button"></sl-icon>
                   ${this.menuItems?.map(
                     menuItem => html`
-                      <sl-menu-item @click=${() => this.#onMenuItemClick(menuItem.tab)} ?disabled=${menuItem.disabled}>
+                      <sl-menu-item
+                        @keydown=${this.#onKeydown}
```

Comment:
Why the keydown event handler? I thought menu-item already does `this.click()` if you're using the keyboard?

Reply:
It was not working properly with link.

Reply:
Just so i understand: using the keyboard with menu-item and pressing enter or space, *doesn't* trigger the click event?

Reply:
My bad, you're right, keydown is unnecessary here in the menu-item :)